app/services/settings.ts

```diff
@@ -156,6 +160,28 @@ export class SettingsService extends StatefulService<ISettingsState> implements
       });
     }
 
+    // We hide the encoder preset and settings if the optimized ones are in used
+    if (categoryName === 'Output' && this.videoEncodingOptimizationService.isUsingEncodingOptimizations()) {
+      const indexSubCategory =
+      settings.indexOf(settings.find((category: any) => {
+        return category.nameSubCategory ===  'Streaming';
+      }));
```

Comment:
This formatting is a little strange.  I would just run Prettier on this file.  The diff might be a little big since we haven't run it on this file before, but that's fine.

Reply:
Done.